src/ostorlab/scanner/handler.py

```diff
@@ -0,0 +1,229 @@
+                    sys.exit(5)
+            except errors.ServiceUnavailableError as e:
+                logger.error("Error in ensure running: %s", e)
+
+    async def subscribe(
+        self,
+        subject: str,
+        cb,
+        queue: str = "",
+        durable_name: Optional[str] = None,
+        start_at: str = "first",
+        max_inflight: int = DEFAULT_MAX_INFLIGHT,
+        manual_acks: bool = True,
+        ack_wait: int = DEFAULT_ACK_WAIT,
+    ):
+        """Start bus subscription.
+
+        subject: Subject for the Bus Streaming subscription.
+        cb: Callback that receive the subject and deserialized Proto message.
+        queue: Queue group.
+        durable_name: Durable connection name.
+        start_at: One of the following options:
+           - 'new_only' (default)
+           - 'first'
+           - 'sequence'
+           - 'last_received'
+           - 'time'
+        max_inflight: Max number of message in flight to client.
+        manual_acks: Toggles auto ack functionality in the subscription callback so that it is implemented by
+         the user instead.
+        ack_wait: How long to wait for an ack before being redelivered previous messages.
+        """
+        self._subjects_cb_map[subject] = cb
+
+        if start_at == "new_only":
+            deliver_policy = js_api.DeliverPolicy.NEW
+        elif start_at == "first":
+            deliver_policy = js_api.DeliverPolicy.ALL
+        elif start_at == "sequence":
+            deliver_policy = js_api.DeliverPolicy.BY_START_SEQUENCE
+        elif start_at == "last_received":
+            deliver_policy = js_api.DeliverPolicy.LAST
+        elif start_at == "time":
+            deliver_policy = js_api.DeliverPolicy.BY_START_TIME
+        else:
+            deliver_policy = js_api.DeliverPolicy.NEW
+
+        sub: js_client.Subscription = await self._js.subscribe(
+            subject=subject,
+            queue=queue,
+            config=js_api.ConsumerConfig(
+                durable_name=durable_name,
+                ack_wait=ack_wait,
+                deliver_policy=deliver_policy,
+                max_ack_pending=max_inflight,
+            ),
+            cb=self._process_message,
+            manual_ack=manual_acks,
+        )
+        inbox_sub = self._nc._subs[sub._id]  # pylint: disable=W0212
+        inbox_sub.pending_bytes_limit = DEFAULT_PENDING_BYTES_LIMIT
+
+    async def _process_message(self, message):
+        try:
+            logger.debug("process received message %s", message)
+            self._last_message_received_time = datetime.datetime.now()
+            request = startAgentScan_pb2.Message()
+            request.ParseFromString(message.data)
+            cb = self._subjects_cb_map[message.subject]
+            await cb(message.subject, request)
+            logger.debug("Acking message for %s", message.subject)
+            await message.ack()
+        except KeyError as e:
+            logger.exception(e)
```

Comment:
Fix the formatting please, also maybe consider using .get() to avoid the whole try catch no ?